pkgs/applications/misc/harsh/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "harsh";
+  version = "0.8.28";
+
+  src = fetchurl {
+    url = "https://github.com/wakatara/harsh/releases/download/v${version}/harsh_Linux_x86_64.tar.gz";
+    sha256 = "0ydz8srxyca1g2wrbvskx69s874rksn2nh4rj54s4xvd5pblyiqq";
+  };
+
+  unpackPhase = ''
+    tar -xf ${src}
+  '';
+
```

Comment:
I think the default `unpackPhase` is sufficient. You can probably just drop this whole part.

Reply:
It actually isn't, as the default unpackPhase throws an error if the tar file doesn't contain a directory